modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/LiferayExperienceCloud/components/SetupLXCModal/components/SetupLXCForm/components/SetupLXCPage/hooks/useSubmitLXCEnvironment.js

```diff
@@ -118,6 +121,39 @@ export default function useSubmitLXCEnvironment(
 						}
 					)
 				);
+
+				if (featureFlags.includes('LPS-181033')) {
+					const adminInfo = [];
+
+					lxcActivationFields?.admins?.map(({email, fullName}) => {
+						const [firstName, ...lastNames] = fullName.split(' ');
+						const lastName = lastNames.join(' ');
+
+						const projectAdminEmailBody = `
+							<strong>First Name -</strong> ${firstName}<br>
+							<strong>Last Name - </strong>${lastName}<br>
+							<strong>Email Address - </strong>${email}
+							<br><br>`;
+
+						adminInfo.push(projectAdminEmailBody);
+					});
```

Comment:
Good job @Aline160 , it looks really good!

I'll try to forward this PR, unless the PR failed, we can send this suggestion later

```suggestion
					const adminInfo = lxcActivationFields?.admins?.map(({email, fullName}) => {
						const [firstName, ...lastNames] = fullName.split(' ');
						const lastName = lastNames.join(' ');

						const projectAdminEmailBody = `
							<strong>First Name -</strong> ${firstName}<br>
							<strong>Last Name - </strong>${lastName}<br>
							<strong>Email Address - </strong>${email}
							<br><br>`;

						return projectAdminEmailBody;
					});
```

Reply:
Done!